packages/components/addon/components/hds/disclosure/index.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="hds-disclosure" ...attributes>
   <div class="hds-disclosure__toggle">
-    {{yield (hash onClickToggle=this.onClickToggle) to="toggle"}}
+    {{yield (hash onClickToggle=this.onClickToggle isActive=this.isActive) to="toggle"}}
```

Comment:
[issue] we should document `isActive` now that is exposed to the consumers 

(see eg. `[:toggle].onClickToggle` in the documentation page for the `Disclosure` component)

Reply:
👍